server/src/main/java/org/apache/cassandra/sidecar/config/KeyStoreConfiguration.java

```diff
@@ -55,7 +55,7 @@ public interface KeyStoreConfiguration
      */
     default boolean reloadStore()
     {
-        return checkInterval().compareTo(SecondBoundConfiguration.ZERO) > 0;
+        return checkInterval() == SecondBoundConfiguration.ZERO;
```

Comment:
this doesn't look correct. We want to reload the store only if the configuration is greater than 0. Not if the configuration is zero.

Reply:
👍 true. it is inverted